e2e/event_test.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"context"
+	"testing"
+
+	"github.com/mesg-foundation/engine/protobuf/acknowledgement"
+	pb "github.com/mesg-foundation/engine/protobuf/api"
+	"github.com/stretchr/testify/require"
+)
+
+func testEvent(t *testing.T) {
+	stream, err := client.EventClient.Stream(context.Background(), &pb.StreamEventRequest{
+		Filter: &pb.StreamEventRequest_Filter{},
+	})
+	require.NoError(t, err)
+	acknowledgement.WaitForStreamToBeReady(stream)
+
+	resp, err := client.EventClient.Create(context.Background(), &pb.CreateEventRequest{
+		InstanceHash: testInstanceHash,
+		Key:          "ping_ok",
+	})
+	require.NoError(t, err)
+
+	event, err := stream.Recv()
+	require.NoError(t, err)
+
+	require.Equal(t, resp.Hash, event.Hash)
+	require.Equal(t, testInstanceHash, event.InstanceHash)
+	require.Equal(t, "ping_ok", event.Key)
```

Comment:
Can you add a check on the event's data?

Reply:
Can you remind me why have you removed `compare` opinion for gogo.protobuf types?

Reply:
i did? i think we should have those helpers (compare, equal, etc..)

Reply:
actually, only process, ownership and execution have `option (gogoproto.equal_all) = true;`
we can definitely set this on all proto